Doc/library/sys.rst

```diff
@@ -523,7 +523,7 @@ always available. Unless explicitly noted otherwise, all variables are read-only
 
    Since :func:`exit` ultimately "only" raises an exception, it will only exit
    the process when called from the main thread, and the exception is not
-   intercepted. Cleanup actions specified by finally clauses of :keyword:`try` statements
+   intercepted. Cleanup actions specified by :keyword:`finally` clauses of :keyword:`try` statements
    are honored, and it is possible to intercept the exit attempt at an outer level.
```

Comment:
Consider wrapping the text to keep the line length down.
```suggestion
   the process when called from the main thread, and the exception is not
   intercepted. Cleanup actions specified by :keyword:`finally` clauses of
   :keyword:`try` statements are honored, and it is possible to intercept the
   exit attempt at an outer level.
```

Reply:
Done.